engine/robot_state.py

```diff
@@ -93,8 +93,8 @@ def __init__(self, **kwargs):
         # If false, only uses GPS and IMU; else, uses EKF
         self.using_ekf = False
         self.is_sim = kwargs.get("is_sim", not is_raspberrypi())
-        self.should_store_data = kwargs.get("should_store_data", False)
-        self.phase = Phase(kwargs.get("phase", Phase.SETUP))
+        self.store_data = kwargs.get("store_data", False)
+        self.phase = Phase(kwargs.get("phase", Phase.TRAVERSE if self.is_sim else Phase.SETUP))
```

Comment:
Is there any setup needed for a simulation or no

Reply:
Thats a good question 🤔  i think there are variables like truthpose in robot_state and some waypoint attributes in mission that are only used in the case of a simulation. 

If we want to make it cleaner we could nest it those attributes under the is_sim flag?? I guess this depends if we only want the matplotlib graph to appear/populate that info only in simulated cases tho. Hmmm not sure but we might need this info for the gui to help us plot a live graph?? 🤔 

do u think its worth?

Reply:
in that case, its probably fine how it is cuz we might want to also have a graph while testing on the robot idk

Reply:
omg we just talked about having a graph on the robot yesterday thats wild